docs/architecture-decisions/2023-06-06_frontend-templates.md

```diff
@@ -13,23 +13,54 @@ AlgoKit v2 aims provide an end-to-end development and deployment experience that
 
 ## Requirements
 
-- **Modularity**: The dApp templating feature should follow guiding AlgoKit principles and expand on approaches already utilized in existing smart contract templates feature. This implies that giving developers flexibility to mix and match different `smart contract` templates with `frontend` templates should serve as key consideration.
```

Comment:
@aorumbayev  These were good, I think it makes sense to keep them in principles since they are a specific reflection for this on how those broader principles apply.

Reply:
I figured since there is a ref directly to a markdown with all of these outlined it may seem redundant but i refined the principles section a bit so should be addressed now.